Build fails when `transparent-inputs` enabled

Build fails due to unresolved import:
```
error[E0412]: cannot find type `Address` in this scope
    --> zcash_client_sqlite/src/lib.rs:2214:20
     |
2214 |         list: Vec<(Address, TransparentAddress, NonHardenedChildIndex)>,
     |                    ^^^^^^^ not found in this scope
```
<details>
<summary>full build log</summary>

```
$ cargo check --features "transparent-inputs"
   Compiling zcash_client_backend v0.21.0 (/home/user/Projects/zcash/work/work/librustzcash/transparent-input-test-build/zcash_client_backend)
   Compiling zcash_client_memory v0.0.0 (/home/user/Projects/zcash/work/work/librustzcash/transparent-input-test-build/zcash_client_memory)
warning: unused imports: `NonHardenedChildIndex` and `std::ops::Range`
   --> zcash_client_backend/src/data_api.rs:108:5
    |
108 |     std::ops::Range,
    |     ^^^^^^^^^^^^^^^
...
113 |         keys::{NonHardenedChildIndex, TransparentKeyScope},
    |                ^^^^^^^^^^^^^^^^^^^^^
    |
    = note: `#[warn(unused_imports)]` on by default

warning: unused import: `BranchId`
  --> zcash_client_backend/src/data_api/wallet/input_selection.rs:20:36
   |
20 |     consensus::{self, BlockHeight, BranchId},
   |                                    ^^^^^^^^

warning: `zcash_client_backend` (lib) generated 2 warnings (run `cargo fix --lib -p zcash_client_backend` to apply 2 suggestions)
    Checking zcash_client_sqlite v0.19.3 (/home/user/Projects/zcash/work/work/librustzcash/transparent-input-test-build/zcash_client_sqlite)
error[E0412]: cannot find type `Address` in this scope
    --> zcash_client_sqlite/src/lib.rs:2214:20
     |
2214 |         list: Vec<(Address, TransparentAddress, NonHardenedChildIndex)>,
     |                    ^^^^^^^ not found in this scope
     |
help: consider importing one of these items
     |
35   + use incrementalmerkletree::Address;
     |
35   + use zcash_address::unified::Address;
     |
35   + use zcash_client_backend::address::Address;
     |
35   + use zcash_client_backend::proto::service::Address;
     |
       and 1 other candidate

warning: unused import: `ops::Range`
 --> zcash_client_sqlite/src/wallet/transparent/ephemeral.rs:2:11
  |
2 | use std::{ops::Range, time::SystemTime};
  |           ^^^^^^^^^^
  |
  = note: `#[warn(unused_imports)]` on by default

warning: unused import: `address::TransparentAddress`
 --> zcash_client_sqlite/src/wallet/transparent/ephemeral.rs:8:5
  |
8 |     address::TransparentAddress,
  |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: unused imports: `BlockHeight` and `self`
  --> zcash_client_sqlite/src/wallet/transparent/ephemeral.rs:13:33
   |
13 | use zcash_protocol::consensus::{self, BlockHeight};
   |                                 ^^^^  ^^^^^^^^^^^

warning: unused import: `AccountRef`
  --> zcash_client_sqlite/src/wallet/transparent/ephemeral.rs:18:5
   |
18 |     AccountRef, AccountUuid,
   |     ^^^^^^^^^^

warning: unused import: `encoding::AddressCodec`
   --> zcash_client_sqlite/src/lib.rs:124:9
    |
124 |         encoding::AddressCodec,
    |         ^^^^^^^^^^^^^^^^^^^^^^

warning: unused import: `zcash_keys::encoding::AddressCodec`
  --> zcash_client_sqlite/src/wallet/transparent/ephemeral.rs:12:5
   |
12 | use zcash_keys::encoding::AddressCodec;
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

warning: irrefutable `if let` pattern
   --> zcash_client_sqlite/src/wallet/transparent.rs:565:16
    |
565 |             if let TransparentAddressSource::Derived { address_index, .. } = meta.source() {
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    |
    = note: this pattern will always match, so the `if let` is useless
    = help: consider replacing the `if let` with a `let`
    = note: `#[warn(irrefutable_let_patterns)]` on by default

For more information about this error, try `rustc --explain E0412`.
warning: `zcash_client_sqlite` (lib) generated 7 warnings
error: could not compile `zcash_client_sqlite` (lib) due to 1 previous error; 7 warnings emitted
```
</details>

--- a/zcash_client_sqlite/src/lib.rs
+++ b/zcash_client_sqlite/src/lib.rs
@@ -130,11 +130,10 @@ use {
 use {
     rusqlite::named_params,
     zcash_client_backend::data_api::{OutputOfSentTx, WalletTest, testing::TransactionSummary},
-    zcash_keys::address::Address,
 };
 
 #[cfg(any(test, feature = "test-dependencies", feature = "transparent-inputs"))]
-use crate::wallet::encoding::KeyScope;
+use {crate::wallet::encoding::KeyScope, zcash_keys::address::Address};
 
 #[cfg(any(test, feature = "test-dependencies", not(feature = "orchard")))]
 use zcash_protocol::PoolType;

